folium/features.py

```diff
@@ -182,11 +197,15 @@ def render(self, **kwargs):
             """).render(this=self, **kwargs)), name=self.get_name())
 
         figure.header.add_child(
-            JavascriptLink('https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js'),  # noqa
+            JavascriptLink(
+                'https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js'),
+            # noqa
```

Comment:
No need for these break lines. Shifting code around makes it hard to review.

Reply:
I think for these I had moved them around to try and keep everything in compliance with the <80 character line style guideline.